src/rsz/test/repair_fanout7.ok

```diff
@@ -252,49 +252,151 @@ Iteration | Removed | Resized | Inserted | Cloned Gates | Pin Swaps |   WNS   |
+     1276 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1277 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1278 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1279 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1280 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1280 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1281 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1282 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1283 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1284 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1285 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1286 |       0 |     614 |      211 |            8 |        10 |  -4.392 | -330.352 | load23/D
+     1289 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1290 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1290 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1291 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1292 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1293 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1294 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1295 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1296 |       0 |     616 |      211 |            8 |        10 |  -4.392 | -330.313 | load23/D
+     1299 |       0 |     618 |      211 |            8 |        10 |  -4.392 | -329.671 | load23/D
+     1300 |       0 |     618 |      211 |            8 |        10 |  -4.392 | -329.671 | load23/D
+    final |       0 |     618 |      211 |            8 |        10 |  -4.392 | -329.671 | load23/D
 -------------------------------------------------------------------------------------------------
-[INFO RSZ-0045] Inserted 70 buffers, 1 to split loads.
-[INFO RSZ-0041] Resized 674 instances.
+[INFO RSZ-0045] Inserted 89 buffers, 1 to split loads.
+[INFO RSZ-0041] Resized 618 instances.
 [INFO RSZ-0043] Swapped pins on 10 instances.
-[INFO RSZ-0049] Cloned 7 instances.
+[INFO RSZ-0049] Cloned 8 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
 Repair timing output passed/skipped equivalence test
 worst slack -4.39
```

Comment:
For a future PR: it would be good to include tns in these tests, as it stands you can't see this making any improvement in the diffs as wns is unchanged.

Reply:
Actually, this test uses -verbose option, so TNS progress is in the log including start TNS and end TNS.